content/docs/pipelines/troubleshooting.md

```diff
@@ -0,0 +1,86 @@
++++
+title = "Troubleshooting"
+description = "Finding and fixing problems in your Kubeflow Pipelines deployment"
+weight = 65
++++
+
+This page presents some hints for troubleshooting specific problems that you
+may encounter.
+
+## Diagnosing problems in your Kubeflow Pipelines environment
+
+For help diagnosing environment issues that affect Kubeflow Pipelines, run
+the [`kfp diagnose_me` command-line tool](/docs/pipelines/sdk/sdk-overview/#kfp-cli-tool).
+
+The `kfp diagnose_me` CLI reports on the configuration of your local
+development environment, Kubernetes cluster, or Google Cloud environment.
```

Comment:
Nit: ["Use this command to help resolve issues like the following:"](https://developers.google.com/style/colons#introductory-phrase-preceding-colon)